proposed/extended-coding-style-guide.md

```diff
@@ -82,9 +82,9 @@ each word is capitalised including the very first letter.
 
 ### 2.2 Files
 
-All PHP files MUST use the Unix LF (linefeed) line ending.
+All PHP files MUST use the Unix LF (linefeed) line ending, applying to every line in the file.
 
-All PHP files MUST end with a single line, containing only a single newline (LF) character.
+All PHP files MUST end with a non-blank line, terminated with a single LF like all other lines in the file.
```

Comment:
I would suggest to remove `like all other lines in the file.`, for two reasons:
1. `like` always provide room for interpretation (the word _like_ is about similarity).
2. `like all other lines in the file` forces the reader to reread every single rule about `lines in the file`.